aea/cli/add.py

```diff
@@ -206,8 +208,10 @@ def fetch_item_mixed(
         package_path = find_item_locally_or_distributed(
             ctx, item_type, item_public_id, dest_path
         )
-    except click.ClickException:
-        logger.debug("Fetch from local registry failed, trying remote registry...")
+    except click.ClickException as e:
+        logger.debug(
+            f"Fetch from local registry failed (reason={str(e)}), trying remote registry..."
```

Comment:
Otherwise it's impossible to know what's wrong with a local package in mixed mode